metar.ts

```diff
@@ -64,3 +68,15 @@ export default async (msg: Message) => {
 	}
 	await sendMetarMessage(metar, msg);
 };
+
+function calcDA(icao_id:string,temp:number,alt_set:number) { // Calculate density altitude for ZAB airfields
+    icao_id = icao_id.toUpperCase();
```

Comment:
camelCase is the JS style for variables: `icaoId`

Reply:
Done